homeassistant/components/homewizard/__init__.py

```diff
@@ -37,7 +39,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: HomeWizardConfigEntry) -
             clientsession=async_get_clientsession(hass),
         )
 
-        await async_check_v2_support_and_create_issue(hass, entry)
+        if is_battery:
+            await async_check_v2_support_and_create_issue(hass, entry)
```

Comment:
This combi disables the repair issue creation entirely. Code itself can be removed if needed, but I plan to enable everything in the next release.

What is common practice?